BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
+
+// REST API to get exchange rates based on input currencies
+app.MapPost("/", service.GetExchangeRates);
+
+app.Run();
+
+public struct CnbRatesResponse
+{
+    [JsonPropertyName("rates")] public List<Rate> Rates { get; set; }
+}
+
+public struct Rate
+{
+    [JsonPropertyName("currencyCode")] public string CurrencyCode { get; set; }
+    [JsonPropertyName("rate")] public float RateValue { get; set; }
```

Comment:
```suggestion
    [JsonPropertyName("rate")] public float Rate { get; set; }
```
As per the previous command, we rename the struct so can call this property "Rate" for clarity.